src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public AbstractFigure getRandomFigure() {
+        int randomNumber = random.nextInt(5);
+        switch (randomNumber) {
+            case 0:
+                return createSquare();
+            case 1:
+                return createRightTriangle();
+            case 2:
+                return createRectangle();
+            case 3:
+                return createIsoscelesTrapezoid();
+            default:
+                return createCircle();
+        }
+    }
+
+    public Circle getDefaultFigure() {
+        return new Circle(Color.WHITE.name().toLowerCase(), 10);
```

Comment:
Please, make these constants `Color.WHITE.name().toLowerCase()` and `10` (`DEFAULT_COLOR`, `DEFAULT_RADIUS`)